app/src/main/java/org/sopt/androidstudy/data/model/UserInfo.kt

```diff
@@ -0,0 +1,29 @@
+package org.sopt.androidstudy.data.model
+
+import android.os.Parcel
+import android.os.Parcelable
+
+data class UserInfo(
+    val email: String?
+) : Parcelable {
```

Comment:
```suggestion

@Parcelize
data class UserInfo(
    val email: String?
) : Parcelable
```

@Parcelize 어노테이션을 추가하면 9번 라인에서 코드 작성을 마칠 수 있지~~ ✌️

Reply:
최고최고!!! 반영하겠습니다 :) 🥰